catalog/view/theme/default/template/extension/payment/razorpay_subscription_info.twig

```diff
@@ -0,0 +1,207 @@
+                                {% if plan.entity_id == subscription_details.plan_entity_id %}
+                                  selected
+                                {% endif %}
+                        >{{ plan.plan_name }}</option>
+                      {% endfor %}
+                    </select>
+                  </div>
+                  <div class="form-group">
+                    <label class="control-label">{{ text_qty }}</label>
+                    <input type="text" name="qty" value="{{ subscription_details.qty }}" placeholder="{{ text_qty }}" id="qty" class="form-control" />
+                  </div>
+                    <input type="hidden" name="subscriptionId" value="{{ subscription_details.subscription_id }}"  id="subscription_id" class="form-control" />
+                    <input type="submit" value="{{ update_plan }}" id="plan-update" class="btn btn-primary" />
+                  <input type="button" value="{{ button_cancel }}" class="btn btn-default" id="cancel" style="margin-left: 20px;" />
+
+                </form>
+
+                </td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+      <h3>{{ text_invoice }}</h3>
+      <div class="table-responsive">
+        <table class="table table-bordered table-hover">
+          <thead>
+            <tr>
+              <td class="text-left">{{ text_date }}</td>
+              <td class="text-left">{{ text_invoice_id }}</td>
+              <td class="text-left">{{ text_amount }}</td>
+              <td class="text-left">{{ text_addon }}</td>
+              <td class="text-left">{{ text_status }}</td>
+              <td class="text-left">{{ text_total_amount }}</td>
+            </tr>
+          </thead>
+          <tbody>
+
+          {% if items is not empty %}
+
+            {% if (subscription_details.next_charge_at != subscription_details.end_at)  and (subscription_details.status != "cancelled") and  (subscription_details.status != "created" ) and (subscription_details.remaining_count != 0) %}
+              <tr>
+                <td class="text-left">{{ items.0.billing_start | date('M d, Y') | default("--") }}</td>
+                <td class="text-left">--</td>
+                <td class="text-left">{{  items.0.line_items.0.net_amount/100 }}
+                  <br/><small>{{ items.0.line_items.0.quantity }} x {{ items.0.line_items.0.unit_amount/100 }} per unit</small>
+                </td>
+                <td class="text-left">--</td>
+                <td class="text-left">Next Due</td>
+                <td class="text-left">{{  items.0.line_items.0.net_amount/100 }}</td>
+              </tr>
+              {% else %}
+                {% for item in items %}
+                  <tr>
+                    <td class="text-left">{{ item.billing_start | date('M d, Y') | default("--") }}</td>
+                    <td class="text-left">{{ item.id }}</td>
+                    <td class="text-left">{{  item.line_items.0.net_amount/100 }}
```

Comment:
add invoice link to the id

Reply:
@sonyy-m Invoice details are not required for now so I have not add the link for the same